examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
and this is what apply_proj is calling under the hood (at least, the most similar part to what was hard-coded):

```python
        _projector, info = setup_proj(deepcopy(self.info), activate=True,
                                      verbose=self.verbose)
        # let's not raise a RuntimeError here, otherwise interactive plotting
        if _projector is None:  # won't be fun.
            logger.info('The projections don\'t apply to these data.'
                        ' Doing nothing.')
            return self
        self._projector, self.info = _projector, info
        if isinstance(self, _BaseRaw):
            if self.preload:
                print('yep!!!!')
                self._data = np.dot(self._projector, self._data)
```
why does one use `make_projector_info` while the other uses `setup_proj`? It seems there's something different going on in these two cases, but should there be? Maybe @Eric89GXL has thoughts?

(either way, I gotta get to bed so I'll address this tomorrow AM)